Renovate Config does not update biome schema version

TLDR: Please add the Renovate preset [customManagers:biomeVersions](https://docs.renovatebot.com/presets-customManagers/#custommanagersbiomeversions) when renovate is configured and biome is used :)

More context version:
We use projen for a cdk library and have configured to use biome and have configured renovate (with lock file maintenance).

Each time biome is updated (for example with biome itself, update lock file maintenance or bump all non major dependencies taks), our GitHub Actions Pipeline fails on the "Fail build on mutation task" with an error message similar to:

```
--- a/biome.jsonc
+++ b/biome.jsonc
@@ -46,5 +46,5 @@
     "enabled": true,
     "useIgnoreFile": true
   },
-  "$schema": "node_modules/.pnpm/@biomejs+biome@2.4.6/node_modules/@biomejs/biome/configuration_schema.json"
-}
\ No newline at end of file
+  "$schema": "node_modules/.pnpm/@biomejs+biome@2.4.15/node_modules/@biomejs/biome/configuration_schema.json"
+}
Error: Process completed with exit code 1.
```

The reason is, that by default Renovate does not update the `$schema` in the `biome.jsonc`.
However there is a Renovate preset [customManagers:biomeVersions](https://docs.renovatebot.com/presets-customManagers/#custommanagersbiomeversions) available from renovate, which solves that.

--- a/src/renovatebot.ts
+++ b/src/renovatebot.ts
@@ -213,18 +213,27 @@ export class Renovatebot extends Component {
       renovateIgnore.push(...new Set(reusableWorkflows));
     }
 
+    const extendsPresets = [
+      ":preserveSemverRanges",
+      "config:recommended",
+      "group:allNonMajor",
+      "group:recommended",
+      "group:monorepos",
+    ];
+
+    // Add Biome custom manager if Biome is configured
+    // Lazy require to avoid circular dependency (renovatebot -> biome -> project -> renovatebot)
+    const { Biome } = require("./javascript/biome/biome"); // eslint-disable-line @typescript-eslint/no-require-imports
+    if (Biome.of(this._project)) {
+      extendsPresets.push("customManagers:biomeVersions");
+    }
+
     return {
       labels: this.labels,
       schedule: this.scheduleInterval,
       minimumReleaseAge: this.minimumReleaseAge,
       minimumReleaseAgeBehaviour: this.minimumReleaseAgeBehaviour,
-      extends: [
-        ":preserveSemverRanges",
-        "config:recommended",
-        "group:allNonMajor",
-        "group:recommended",
-        "group:monorepos",
-      ],
+      extends: extendsPresets,
       packageRules: [
         {
           matchDepTypes: ["devDependencies"],

--- a/test/renovatebot.test.ts
+++ b/test/renovatebot.test.ts
@@ -1,5 +1,6 @@
-import { synthSnapshot, TestProject } from "./util";
 import { DependencyType, RenovatebotMinimumReleaseAgeBehaviour } from "../src";
+import { synthSnapshot, TestProject } from "./util";
+import { TypeScriptProject } from "../src/typescript";
 
 describe("renovatebot", () => {
   test("renovatebot: true creates renovatebot configuration", () => {
@@ -205,5 +206,38 @@ describe("renovatebot", () => {
 
     // THEN
     expect(p.tryFindObjectFile("renovate.json5")).toBeDefined();
+  });
+
+  test("renovatebot: adds customManagers:biomeVersions when biome is configured", () => {
+    // GIVEN
+    const p = new TypeScriptProject({
+      name: "test-project",
+      defaultReleaseBranch: "main",
+      renovatebot: true,
+      biome: true,
+    });
+
+    // WHEN
+    const snapshot = synthSnapshot(p);
+
+    // THEN
+    expect(snapshot["renovate.json5"].extends).toContain(
+      "customManagers:biomeVersions",
+    );
+  });
+
+  test("renovatebot: does not add customManagers:biomeVersions when biome is not configured", () => {
+    // GIVEN
+    const p = new TestProject({
+      renovatebot: true,
+    });
+
+    // WHEN
+    const snapshot = synthSnapshot(p);
+
+    // THEN
+    expect(snapshot["renovate.json5"].extends).not.toContain(
+      "customManagers:biomeVersions",
+    );
   });
 });

